src/GitTools.Core/GitTools.Core.csproj

```diff
@@ -19,36 +19,36 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\output\debug\NET45\GitTools.Core\</OutputPath>
+    <OutputPath>..\..\output\Debug\GitTools.Core\net45\</OutputPath>
```

Comment:
I am structuring similar to NuGet, package then the lib. We will likely end up having multiple build targets